model/src/main/java/de/evoila/cf/broker/model/ServiceInstance.java

```diff
@@ -72,6 +72,10 @@
     @JsonIgnore
     private String usergroup;
 
+    @JsonSerialize
+	@JsonProperty("allow_context_update")
+    private boolean allowContextUpdates;
```

Comment:
Is this field in the right spot? How do you fill it and when? And there are no changes to the service definition class, where it definitely should be added!

Reply:
This is wrong. Will fix it immediately.